### DISCUSSION TITLE
Last-active tab focus when switching workspaces

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#838] [Ideas]
### What feature would you like to see?

Current Behavior (v1.0.0-a.27):
When switching workspaces, Zen browser focuses on the first tab opened in that workspace.

Desired Behavior:
When switching workspaces, automatically focus on the last active tab in that workspace.

Rationale:
This would save time and hassle by not having to search for the last-used tab every time when switching workspaces.

### What platform are you seeing the problem on?

macOS

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/427_

**Comment by Vaelek** [comment]
This occurs still today on Windows as well. This is incredibly annoying for someone that jumps between workspaces quite frequently.
